.eslintrc.js

```diff
@@ -320,16 +320,13 @@ module.exports = {
         /**
          * testing-library
          */
+        "testing-library/await-async-queries": "error",
         "testing-library/prefer-user-event": "error",
-        "testing-library/no-wait-for-empty-callback": "error",
-        "testing-library/no-wait-for-multiple-assertions": "error",
-        "testing-library/no-wait-for-side-effects": "error",
-        "testing-library/no-wait-for-snapshot": "error",
+        // TODO(jeremy): It might be nice to enable
+        // "testing-library/prefer-explicit-assert": "error",
```

Comment:
I'm in favor of this!

Reply:
I'll put up a separate PR and see how disruptive it is, and if we like it.